packages/data/tests/unit/adapters/bard-facts-v2-test.js

```diff
@@ -0,0 +1,965 @@
+import { module, test } from 'qunit';
```

Comment:
It would be nice if the test was also typescript, but might be a lot of work to go back and redo it